docs/docs/FAQ/Machine-Learning-FAQ.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 6
+---
+# Machine Learning
+
+### What is the technology that Immich use for machine learning?
+
+- For search function (smart search) - Immich uses Clip Models, for more information about Clip and its capabilities can be [read here](https://openai.com/research/clip) and [experience here](https://huggingface.co/models?pipeline_tag=zero-shot-image-classification&sort=trending).
```

Comment:
In fact it can give people another point of view on how different models react to different images (if someone wants to test a model that is more trained to "recognize" types of vehicles)

Reply:
The models in that link can't actually be used for Immich, so I think it will just lead to confusion. The answer here should just be about what Immich actually does.